src/workload/paragraph/GraphTerminal.cc

```diff
@@ -70,18 +70,18 @@ GraphTerminal::GraphTerminal(const std::string& _name, const Component* _parent,
   // with log file provided in a similar way to graph file
   std::unique_ptr<paragraph::Logger> logger = nullptr;
   if (!_settings["log_file"].is_null()) {
-      auto logger_format =
-          absl::ParsedFormat<'d'>::New(
-              _settings["log_file"].get<std::string>());
-      assert(logger_format);
-      CHECK_OK_AND_ASSIGN(logger, paragraph::Logger::Create(
-            absl::StrFormat(*logger_format, id_), false));
+    auto logger_format =
+        absl::ParsedFormat<'d'>::New(_settings["log_file"].get<std::string>());
+    assert(logger_format);
+    CHECK_OK_AND_ASSIGN(
+        logger,
+        paragraph::Logger::Create(absl::StrFormat(*logger_format, id_), false));
   }
 
   // Constructs the scheduler
-  CHECK_OK_AND_ASSIGN(std::unique_ptr<paragraph::GraphScheduler> scheduler,
-                      paragraph::GraphScheduler::Create(graph_.get(),
-                                                        std::move(logger)));
+  CHECK_OK_AND_ASSIGN(
```

Comment:
Add `"log_file": "graph.%d.csv"` to config/ring_iq_paragraph.json so that this functionality is at least exercised in the examples.